chart/validator-plugin-oci/templates/deployment.yaml

```diff
@@ -22,33 +22,52 @@ spec:
       annotations:
         kubectl.kubernetes.io/default-container: manager
     spec:
+      {{- if .Values.proxySecretName }}
+      initContainers:
+      - name: init-certs
+        image: ubuntu
+        command: ["/bin/bash", "-c"]
+        args: ["apt-get update && apt-get install -y ca-certificates && update-ca-certificates && cp -r /etc/ssl/certs/* /certs/"]
+        volumeMounts:
+        - name: ca-pemstore
+          mountPath: "/usr/local/share/ca-certificates/{{ .Values.proxySecretName }}.crt"
+          subPath: {{ .Values.proxySecretName }}.crt
```

Comment:
:wrench: This assumes that `.Values.proxySecretName` includes the certificate data under a key whose name matches the Secret name, e.g.:
```yaml
apiVersion: v1
kind: Secret
metadata:
  name: my-ca-cert
data:
  my-ca-cert.crt : |
    --- BEGIN CERT ----
    ...
```
If it works, I think you'd be better off without the `subPath`:
```yaml
- name: ca-pemstore
  mountPath: "/usr/local/share/ca-certificates/"
  readOnly: false
```

Otherwise you'll need to ask the user for the name of the correct key within the Secret.

Reply:
it works without the subPath 🥳